test-data/unit/check-classes.test

```diff
@@ -5930,6 +5930,90 @@ class B:
 class C(A, B): pass
 [out]
 
+[case testBinaryMagicMethods]
+# flags: --warn-return-any
+# See also check-warnings.test [case testReturnAnyForNotImplementedInBinaryMagicMethods]
+from typing import Any
+class A:
+    def __init__(self, x: Any) -> None:
+        self.x = x
+    def __abs__(self) -> 'A': return NotImplemented  # E: Returning Any from function declared to return "A"
+    def __bool__(self) -> bool: return NotImplemented  # E: Returning Any from function declared to return "bool"
+    def __call__(self, *args, **kwargs) -> 'A': return NotImplemented  # E: Returning Any from function declared to return "A"
+    def __ceil__(self) -> 'A': return NotImplemented  # E: Returning Any from function declared to return "A"
+    def __complex__(self) -> 'A': return NotImplemented  # E: Returning Any from function declared to return "A"
+    def __del__(self) -> None: return NotImplemented  # E: Returning Any from function declared to return "None"
+    def __delattr__(self, other: Any) -> None: return NotImplemented  # E: Returning Any from function declared to return "None"
+    def __delete__(self, other: Any) -> None: return NotImplemented  # E: Returning Any from function declared to return "None"
+    def __delitem__(self, other: Any) -> None: return NotImplemented  # E: Returning Any from function declared to return "None"
+    def __dir__(self, other: Any) -> Sequence_str: return NotImplemented  # E: Returning Any from function declared to return "Sequence_str"
+    def __enter__(self, other: Any) -> None: return NotImplemented  # E: Returning Any from function declared to return "None"
+    def __exit__(self, other: Any) -> None: return NotImplemented  # E: Returning Any from function declared to return "None"
+    def __float__(self) -> 'A': return NotImplemented  # E: Returning Any from function declared to return "A"
+    def __floor__(self) -> 'A': return NotImplemented  # E: Returning Any from function declared to return "A"
+    def __get__(self, instance: Any, other: Any) -> str: return NotImplemented  # E: Returning Any from function declared to return "str"
+    def __getitem__(self, other: Any) -> 'A': return NotImplemented  # E: Returning Any from function declared to return "A"
+    def __int__(self) -> 'A': return NotImplemented  # E: Returning Any from function declared to return "A"
+    def __index__(self) -> 'A': return NotImplemented  # E: Returning Any from function declared to return "A"
+    def __inv__(self) -> 'A': return NotImplemented  # E: Returning Any from function declared to return "A"
```

Comment:
Again, I think that this isn't a magic method (but `__invert__` is)?

Reply:
https://docs.python.org/3/library/operator.html
seems to show `__inv__` and `__invert__` as synonyms.
But by experiment, `__inv__` doesn't work (either in Python2.7 or Python3.x).

So, is this a bug in the documentation or is there some use of `__inv__` that I'm not aware of?

Reply:
It looks like `__inv__` is specific to the `operator` module, so it doesn't need any special casing.